Source/Tools/FEXServer/FDCountWatch.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+#include <sys/types.h>
+
+namespace FDCountWatch {
```

Comment:
Should probably have a docstring what exactly is being watched here and how and why.

Reply:
Added a comment block about why we need to watch FDs.